src/utils/deploy/hash-fns.mjs

```diff
@@ -110,7 +117,11 @@ const hashFns = async (
     assetType: 'function',
     normalizedPath: path.basename(functionPath, path.extname(functionPath)),
     runtime,
+    displayName,
   }))
+  const fnConfig = functionZips
```

Comment:
I think in the future, and especially once we add more properties to this object, it would be nice to abstract this into a separate method and have a proper validation mechanism. For example, in here you're ensuring that you're not passing any unknown properties, but you're not doing anything to ensure that `display_name` has the right type.